packages/core/tocco-ui/src/Select/Select.js

```diff
@@ -46,6 +46,10 @@ const Select = ({
 }) => {
   const selectComponent = useRef(null)
   const selectWrapper = useRef(null)
+  const getWrapperWidth = () => selectWrapper.current?.clientWidth || 300
+  const getWrapperHeight = () => selectWrapper.current?.clientHeight || 35
+  const [wrapperWidth, setWrapperWidth] = useState(getWrapperWidth)
+  const [wrapperHeight, setWrapperHeight] = useState(getWrapperHeight)
```

Comment:
Is the initial state working properly?
Shouldn't it be `useState(getWrapperWidth())`?